### DISCUSSION TITLE
Get involved in the formation of the new Git and GitHub CoP!

### DISCUSSION THREAD

**Discussion by andrewcoughtrie** [MetOffice/simulation-systems#84] [Announcements]
Hi Everyone,

With the ever-increasing use of Git and GitHub across Science, Technology, and external partners we thought it would be interesting and useful to form a Git and GitHub Community of Practice (CoP) to bring together knowledge, experience, and interest in what features are available, how people can and have been using them, and to disseminate this information for the benefit of others. The CoP itself will also be a great resource for those new to Git and GitHub to learn about their use and how it can benefit them.

**We ask those interested in participating in a session (1.5-2 hours long max) discussing and agreeing the purpose and vision of the CoP, which will be scheduled in November, to react to this message by the end of Friday, November 3, 2023.** 

Andy Coughtrie (Science IT, Foundation Science), 
Emma Hogan (Science IT, Foundation Science), 
Laura Dreyer (AVD, Technology), 
Henry Wright (AVD, Technology)

**Comment by andrewcoughtrie** [comment]
Hi all, 
I've just realised I can't actually see who likes the post so if you could reply to this message so I can at least see the names of the people who are interested.

Thanks

Andy

**Reply by JorgeBornemann** [reply]
Sorry, I have just been made aware if this. Can I still sign up?  (Jorge Bornemann, NIWA, Aotearoa New Zealand)

**Comment by ccarouge** [comment]
Hi Andy,

I just saw this now as well. I'd be happy to join in: Claire Carouge, ACCESS-NRI, Australia.

**Comment by KellyOMe** [comment]
Hi Andy, I have also only just seen this, Please can I join too?  Kelly O'Meara, AI4NWP, Exeter, UK

**Comment by hiker** [comment]
I'm a bit late, if possible can I still join? Joerg Henrichs, BOM.
